docs/asff.md

```diff
@@ -36,16 +36,17 @@ Here is an example IAM Policy that you can attach to your EKS node group's IAM R
 }
 ```
 
-## Configure and rebuild kube-bench
+## Build kube-bench
 
-You will need to download, build and push the kube-bench container image to your ECR repo as described in Step 3 of the [EKS instructions][eks-instructions], except that before you build the container image, you need to edit `cfg/eks-1.0/config.yaml` to specify the AWS account, AWS region, and the EKS Cluster ARN.
+You will need to download, build and push the kube-bench container image to your ECR repo as described in Step 3 of the [EKS instructions][eks-instructions].
```

Comment:
```suggestion
You will need to download, build and push the kube-bench container image to your ECR repo as described in Step 3 of the [EKS instructions][eks-instructions]. Before you build the container image, you need to edit the job configuration.
```

Reply:
@lizrice Not sure on adding that line as the job configuration doesn't need to be added before the container image has been built / pushed. But now with the v0.5.0 release, this whole block is probably redundant now as the image doesn't need to be custom built anymore - so maybe we could remove this section entirely. What do you think?

Reply:
That's true, let's get rid of this part